src/main/resources/changelog-master.xml

```diff
@@ -67,4 +67,5 @@
     <include file="changesets/changelog-consent-62.0.xml" relativeToChangelogFile="true"/>
     <include file="changesets/changelog-consent-63.0.xml" relativeToChangelogFile="true"/>
     <include file="changesets/changelog-consent-64.0.xml" relativeToChangelogFile="true"/>
```

Comment:
When I check out the PR, these files aren't there. I think this PR needs to be rebased to get https://github.com/DataBiosphere/consent/pull/1002

Reply:
Resolved!